node/cn/api.go

```diff
@@ -283,6 +284,11 @@ func NewPublicDebugAPI(cn *CN) *PublicDebugAPI {
 	return &PublicDebugAPI{cn: cn}
 }
 
+func (api *PublicDebugAPI) GetRocksDBProperty(dt database.DBEntryType, name string) string {
+	dbm := api.cn.ChainDB()
+	return dbm.GetProperty(dt, name)
```

Comment:
How about adding a more general API such as `GetDBProperty`? Because Klaytn already has a similar API called `ChaindbProperty` and it provides properties of levelDB.

Reply:
I will leave the `ChaindbProperty` and `ChaindbCompaction`, but add new `GetDBProperty` for rocksdb and leveldb.